server/app/views/applicant/ProgramCardsSectionFragment.html

```diff
@@ -26,6 +26,11 @@
 >
   <div class="usa-card__container">
```

Comment:
(note) for the future. When you get the final mocks we probably want to make sure that we have an max-width set on the card to the width isn't unbounded.

Reply:
SGTM. I put some placeholder values in until the final mocks are available.